src/main/java/frc/robot/subsystems/vision/Vision.java

```diff
@@ -0,0 +1,35 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems.vision;
+
+import frc.robot.subsystems.drive.DrivingDeltas;
+import frc.robot.subsystems.vision.Limelight;
+import frc.robot.Constants;
+import frc.robot.subsystems.Lights;
+
+/**
+ * Add your docs here.
+ */
+public class Vision {
```

Comment:
Like @nbgray suggested, let's get rid of this file.

Reply:
Removed